components/wgputoycontroller.tsx

```diff
@@ -550,6 +559,12 @@ const WgpuToyController = props => {
                 ComputeEngine.getInstance().setMouseClick(
                     e instanceof MouseEvent ? e.button + 1 : 1
                 );
+
+                // Add pointer lock on left mouse click
```

Comment:
I don't think shadertoy uses a pointer lock either?

Reply:
This PR still in progress, need to rework it.